### DISCUSSION TITLE
Creating PR betwen git repos when github is unaware they are forks of each other

### DISCUSSION THREAD

**Discussion by xloem** [community/community#22631] [New to GitHub]
<p>Hi,</p>
<p>I’m trying to create a PR for lirtlsdr/librtlsdr from xloem/rtl-sdr, but github is not aware they are forks because they originated from differing imports of the project; they do share historical git commits.</p>
<p>Is there any way to link these projects in github’s system, so I can make the PR?</p>
<p>Thanks.</p>

**Comment by mpboom** [comment]
<p>What about forking the lirtlsdr/librtlsdr repo to your account, then applying your changes locally there, then making a PR?</p>

**Comment by xloem** [comment]
<p>Hi Mark,</p>
<p>That’s the solution I ended up using.  There is a larger ecosystem problem, though, where this project has a handful of different imports into github, many of which have significant activity and forking of their own going on.  It’s hard to participate in them together.</p>
<p>It sounds like there is no way at the moment to work towards resolving the situation?</p>
<p>Karl</p>

**Comment by mpboom** [comment, marked as answer]
<p>Probably not. But as said, you can easily just fork it yourself, clone it then force push your changes. However, the moderators could write this down as a feature reqeuest.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/mpboom">@mpboom</a>, thanks for the mention, and thank you and <a class="mention" href="/xloem">@xloem</a>. We appreciate the feedback. I’ll pass it along to the appropriate teams, but I can’t make any promises!</p>

**Comment by xloem** [comment]
<p>Thanks so much all =)</p>
